core/src/test/java/com/datastax/oss/driver/internal/core/util/concurrent/CapturingTimer.java

```diff
@@ -51,12 +49,7 @@ public Timeout newTimeout(TimerTask task, long delay, TimeUnit unit) {
    * tell from the returned Timeout itself.
    */
   public CapturedTimeout getNextTimeout() {
-    try {
-      return timeoutQueue.poll(100, TimeUnit.MILLISECONDS);
-    } catch (InterruptedException ie) {
-      fail("Unexpected interruption", ie);
-      throw new AssertionError();
-    }
+    return timeoutQueue.poll();
```

Comment:
We don't need to wait anymore here. Since write promises are completed on the calling thread, the scheduled timeout is either already there, or will never be.